polaris-react/src/styles/shared/_buttons.scss

```diff
@@ -206,8 +206,8 @@
 }
 
 @mixin plain-button-backdrop {
-  padding: var(--p-space-05) 5px;
-  margin: calc(-1 * var(--p-space-05)) -5px;
```

Comment:
Chromatic didn't flag this but here's the before and after (difference is `1px` padding on left and right of grey backdrop):
**After**
<img width="170" alt="button-plain-left-right-padding-new" src="https://user-images.githubusercontent.com/21976492/163887116-190d1f62-1aa0-49e2-b62c-dc057dbdf21a.png">
**Before**
<img width="170" alt="button-plain-left-right-padding-old" src="https://user-images.githubusercontent.com/21976492/163887119-24464f9c-b36d-44b2-aa94-4d771a655fe4.png">